.github/workflows/au4_build_linux.yml

```diff
@@ -90,6 +90,16 @@ jobs:
       uses: turtlebrowser/get-conan@main
     - name: Register gcc problem matcher
       run: echo "::add-matcher::.github/problem_matchers/gcc.json"
+
+    - name: Install Qt
+      uses: jurplel/install-qt-action@v4
+      with:
+        version: '6.9.1'
```

Comment:
6.9.2 has been released this week, so you might want to start with that version immediately

Reply:
Will try :)